src/store/index.js

```diff
@@ -22,25 +24,59 @@ export default new Vuex.Store({
       state.movies = { ...movies };
       state.searchQuery = '';
     },
+    GET_ACTORS(state, actors) {
```

Comment:
It says "GET", but you're setting a variable.

Why did you name a mutation with "GET"?